### PR TITLE
Fixing the Height

### PR DESCRIPTION
**Pull request by hanarimawi**
adjusted the base components to get rid of the bottom whitespace

### CODE REVIEW

File: frontend/src/styles/styles.scss [hunk 1]
```diff
@@ -76,6 +76,7 @@
   height: 100%;
   background-size: cover;
   background-position: center;
+  height: 100%;
```
Comment (angadgarg25) [inline frontend/src/styles/styles.scss]: do we need this here twice? its on line 76 as well

File: frontend/src/styles/results.scss [hunk 2]
```diff
@@ -94,6 +94,7 @@
 .backgroun {
```
Comment (angadgarg25) [inline frontend/src/styles/results.scss]: is this a typo

**Review by angadgarg25 — CHANGES_REQUESTED** [review]

File: frontend/src/styles/styles.scss (hunk 1 above)
Comment (HRimaw2) [reply frontend/src/styles/styles.scss]: u right

File: frontend/src/styles/results.scss (hunk 2 above)
Comment (HRimaw2) [reply frontend/src/styles/results.scss]: i don't know it wasn't mine

**Pull request merged**